docs/source/team.ini

```diff
@@ -0,0 +1,134 @@
+[a]
+# A section for short links
+icl: http://www.imperial.ac.uk
+gh: https://github.com
+
+[dev-institution]
+# Use format
+# institution: institution_website
+#
+# then complete a section with:
+#
+# [institution]
+# department: website
+Imperial College London: http://www.imperial.ac.uk
+University of Oxford: https://www.ox.ac.uk
+Baylor University: https://www.baylor.edu
+University of Washington: https://www.washington.edu
+
```

Comment:
These are current institutions, I think there should be former institutions, so that Durham are acknowledged (they did pay me for four years).

Reply:
Oh, I see that happens below.